src/cpptools.ts

```diff
@@ -113,6 +113,10 @@ function parseTargetArch(target: string): Architecture {
     return 'arm64';
   } else if (target.indexOf('arm') >= 0 || is_arm_32(target)) {
     return 'arm';
+  } else if (target.indexOf('i686') >= 0) {
+    return 'x86';
+  } else if (target.indexOf('amd64') >= 0 || target.indexOf('x86_64')) {
```

Comment:
Should `target.indexOf('x86_64')` have a check `>= 0`?

Reply:
Yes definitely. Thanks for catching that.